server/routes/viewModels/taskLists/caTasks.js

```diff
@@ -182,8 +181,7 @@ module.exports = {
           validAddress &&
             !['caToDm', 'caToDmRefusal', 'caToRo'].includes(allowedTransition) &&
             !dmRefused &&
-            !postponed &&
-            !useCvlForLicenceCreation,
+            !postponed,
```

Comment:
so.. now show task for these CVL cases?

Reply:
If they are CVL, the task will now be shown with a label and no continue button (provided they meet the other checks) and if HDC, the task will show without a label and the continue button.